core/src/main/java/dev/streamx/cli/run/RunCommand.java

```diff
@@ -108,4 +118,30 @@ void onContainerStarted(@Observes ContainerStarted event) {
   private static void print(String x) {
     System.out.println(x);
   }
+
+
+  private void setupRootAuthToken() {
```

Comment:
extract to new class `RunningMeshPropertiesGenerator`

Reply:
extracted